code/modules/mob/living/basic/farm_animals/pony.dm

```diff
@@ -0,0 +1,59 @@
+/mob/living/basic/pony
+	name = "pony"
+	desc = "Look at my horse, my horse is amazing!"
+	icon_state = "pony"
+	icon_living = "pony"
+	icon_dead = "pony_dead"
+	gender = MALE
+	mob_biotypes = MOB_ORGANIC | MOB_BEAST
+	speak_emote = list("neighs", "winnies")
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "kicks"
+	response_harm_simple = "kick"
+	attack_verb_continuous = "kicks"
+	attack_verb_simple = "kick"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	attack_vis_effect = ATTACK_EFFECT_KICK
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	health = 50
+	maxHealth = 50
+	gold_core_spawnable = FRIENDLY_SPAWN
+	blood_volume = BLOOD_VOLUME_NORMAL
+	ai_controller = /datum/ai_controller/basic_controller/pony
+
+/mob/living/basic/pony/Initialize(mapload)
+	AddElement(/datum/element/pet_bonus, "whickers.")
+	AddElement(/datum/element/ai_retaliate)
+	AddElement(/datum/element/ai_flee_while_injured)
+	AddElement(/datum/element/waddling)
+	make_tameable()
+	. = ..()
+
+/mob/living/basic/pony/proc/make_tameable()
+	AddComponent(/datum/component/tameable, food_types = list(/obj/item/food/grown/apple), tame_chance = 25, bonus_tame_chance = 15, after_tame = CALLBACK(src, PROC_REF(tamed)))
+
+/mob/living/basic/pony/proc/tamed(mob/living/tamer)
+	can_buckle = TRUE
+	buckle_lying = 0
+	AddElement(/datum/element/ridable, /datum/component/riding/creature/pony)
+	visible_message(span_notice("[src] snorts happily."))
+
+/datum/ai_controller/basic_controller/pony
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic/ignore_faction(),
+	)
+
+	ai_traits = STOP_MOVING_WHEN_PULLED
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/find_nearest_thing_which_attacked_me_to_flee,
+		/datum/ai_planning_subtree/flee_target,
+		/datum/ai_planning_subtree/target_retaliate,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree
```

Comment:
there's no subtree for speech? when is the horse gonna whinny or neigh or snort or anything like that?

Reply:
this is a misconception about horses.

they're entirely silent unless they're hollering to find each other or making flustered sputtering noises because you aren't carrying the feed bucket to the stall fast enough

source: i take care of horses

Reply:
broadly:

if a horse is making a noise it's because it's lost or pissed at you

the more you know

Reply:
ultimately whatever

Reply:
i think a horse would be very lost on a space station thinking about it

Reply:
i have added speech subtrees and sound effects